scripts/start.sh

```diff
@@ -326,30 +272,39 @@ if [ $SSHD == "true" ]; then
 		chown gvm:gvm -R /data/scanner-ssh-keys
 		chown gvm:gvm -R /usr/local/share/gvm/.ssh
 	fi
-	
 	if [ ! -d /sockets ]; then
 		mkdir /sockets
 		chown gvm:gvm -R /sockets
 	fi
-	
 	echo "gvm:gvm" | chpasswd
-	
 	rm -rf /var/run/sshd
 	mkdir -p /var/run/sshd
-	
-	/usr/sbin/sshd -f /sshd_config -E /usr/local/var/log/gvm/sshd.log
+	if [ ! -f /data/ssh/sshd_config ]; then
+		mv /sshd_config /data/ssh/sshd_config
+	fi
+	${SUPVISD} start sshd
+	if [ "${DEBUG}" == "Y" ]; then
+		${SUPVISD} status sshd
+	fi
 fi
 
-GVMVER=$(su -c "gvmd --version" gvm )
-echo "++++++++++++++++++++++++++++++++++++++++++++++"
-echo "+ Your GVM $GVMVER container is now ready to use! +"
-echo "++++++++++++++++++++++++++++++++++++++++++++++"
+${SUPVISD} start GVMUpdate
+if [ "${DEBUG}" == "Y" ]; then
+	${SUPVISD} status GVMUpdate
+fi
+if [ "${SETUP}" == "1" ]; then
+	${SUPVISD} shutdown
+fi
+GVMVER=$(su -c "gvmd --version" gvm)
+echo "+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
+echo "+     $GVMVER"
+echo "+ Your GVM container is now ready to use!                 +"
+echo "+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
 echo ""
 echo "-----------------------------------------------------------"
 echo "Server Public key: $(cat /etc/ssh/ssh_host_ed25519_key.pub)"
 echo "-----------------------------------------------------------"
-echo ""
-echo "++++++++++++++++"
-echo "+ Tailing logs +"
-echo "++++++++++++++++"
-tail -F /usr/local/var/log/gvm/*
+echo "-----------------------------------------------------------"
+echo "+        Find logs at: /var/log/supervisor/               +"
+echo "+              and at: /usr/local/var/log/gvm/            +"
+echo "==========================================================="
```

Comment:
This would encourage people to dig into their containers to get basic log information, contrary to the current status quo, which is to show all logs into stdout, to allow for `docker logs` to work.

Prefer using supervisor's log facilities to display logs to stdout.

Reply:
That is something I don't like to do, because you get output of all services mixed up and this helps nobody.

This should never ever done in a multi service container! Only if you can separate each log to it's service. What it is not yet done.

Reply:
Understandable, however, this is still a massive behavior change, one that likely will break some workflows relying on log parsing.

This should be pointed out in the first post, as this is a major breaking change that is not linked to the work of switching to alpine.